src/Gui/ViewProvider.cpp

```diff
@@ -104,7 +105,7 @@ ViewProvider::ViewProvider()
     pcRoot->ref();
     pcModeSwitch = new SoSwitch();
     pcModeSwitch->ref();
-    pcTransform  = new SoTransform();
+    pcTransform  = new SoFCTransform();

```

Comment:
```suggestion
    // Using SoFCTransform as a workaround for coin3d/coin#534.
    // When fixed, replace with SoTransform.
    pcTransform  = new SoFCTransform();
```

Reply:
I don't think we need this extra comment here. Once SoFCTransform should be changed back to SoTransform the compiler will tell us anyway all locations.